src/main/java/com/google/sps/servlets/populateLeaderboardServlet.java

```diff
@@ -57,11 +59,21 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
         return;
     }
  
-    ArrayList<User> users = UserUtils.userList((String) userEntity.getProperty("gameId"), datastore);
+    ArrayList<User> userObjects = UserUtils.userList((String) userEntity.getProperty("gameId"), datastore);
+    
+    // translate to JSON for loadLeaderBoard function
+    JSONArray users = new JSONArray();
+    for(User user : userObjects){
+      JSONObject obj = new JSONObject();
```

Comment:
We talked about this already but I mentioned that you could use a java datastructure here and send it through GSON to the javascript but a JSONArray will work too

Reply:
Okay but I love not using GSON. Not using GSON is maybe my favorite part of using JSON.